synapse/handlers/acme.py

```diff
@@ -27,11 +27,11 @@
 
 ACME_REGISTER_FAIL_ERROR = """
 --------------------------------------------------------------------------------
-Failed to register with the ACME provider. This is likely happening because the install
-is new, and ACME v1 has been deprecated by Let's Encrypt and is disabled for installs set
-up after November 2019.
-At the moment, Synapse doesn't support ACME v2. For more info and alternative solution,
-check out https://github.com/matrix-org/synapse/blob/master/docs/ACME.md#deprecation-of-acme-v1
+Failed to register with the ACME provider. This is likely happening because the installation
+is new, and ACME v1 has been deprecated by Let's Encrypt and disabled for
+new installations after November 2019.
```

Comment:
That phrasing seems a bit weird to me but ymmv

Reply:
"ACMEv1 has been ... disabled for new installations after November 2019". It does sound weird.

I want to get this shipped, so just I'm going to:

```suggestion
new installations since November 2019.
```